views/pages/ConfigPage.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { useOutletContext } from 'react-router-dom'
+import ConfigurationsList from '../components/ConfigurationsList'
+
+const ConfigPage = () => {
+  const { user, classes, theme, navigate } = useOutletContext()
+
+  return (
+    <React.Fragment>
```

Comment:
we don't need to wrap this in a fragment.

Reply:
[3b1cb1e](https://github.com/AMP-SCZ/dpdash/pull/317/commits/3b1cb1edcd218a42e840a24e2326028ff4d55fed)